plugins/module_utils/common.py

```diff
@@ -0,0 +1,24 @@
+
+# Copyright: (c) 2022, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+AMAZON_AWS_COLLECTION_NAME = "amazon.aws"
+AMAZON_AWS_COLLECTION_VERSION = "6.0.0-dev0"
```

Comment:
Is there any way we can avoid hardcoding these info?

Reply:
Unfortunately this information isn't available 'client' side, as we don't have direct access to galaxy.yml (only the files we explicitly send over).

Reply:
I did not find a way to avoid hardcoding this information as @tremble mentioned the only place to find it is from `galaxy.yml` file which is not part of the tarball created by Ansible at execution time.